src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -553,9 +556,13 @@ def open(self):
                 kwargs = {}
             self.__file = File(self.__path, open_flag, **kwargs)
 
-    def close(self):
+    def close(self, close_links=False):
```

Comment:
I would add a docstring for ``close_links``. I'm not sure whether the default should be ``close_links=True`` or ``close_links=False``. Setting the default as ``false`` may be safer, because it matched the current behavior, but in practice, the more common case may be to use ``true`` as it ensures that no files remain silently open.

Reply:
Agreed. Default should be `close_links=True` (this was just a test).